test/MOI_wrapper.jl

```diff
@@ -0,0 +1,88 @@
+using Test
```

Comment:
the function names in this file could be improved a bit (maybe `test_sphere_moi()` or so?)
And the file to `test_MOI_wrapper.jl` as well as being added to the the `runtest.jl` :)

Reply:
@blegat you may as well make this a full test module like we normally do? Rather than manually enumerating the `test_` functions.